types/TestResult.js

```diff
@@ -105,14 +139,36 @@ export type TestResult = {|
   testResults: Array<AssertionResult>,
 |};
 
+export type FormattedTestResult = {|
+  message: string,
+  name: string,
+  summary: string,
+  status: 'failed' | 'passed',
+  startTime: number,
+  endTime: number,
+  coverage: any,
+  assertionResults: Array<FormattedTestAssertion>,
+|};
+
+export type FormattedTestResults = AggregatedResult & {
+  numFailedTests: number,
+  numPassedTests: number,
+  numPendingTests: number,
+  numRuntimeErrorTestSuites: number,
+  numTotalTestSuites: number,
+  numTotalTests: number,
+  startTime: number,
+  success: boolean,
+  testResults: FormattedTestResult,
```

Comment:
given that flow doesn't complain and `FormattedTestResult` contains `assertionResults` which is an `Array<FormattedTestAssertion>` I guess this is correct?